test/unit/account/search_test.rb

```diff
@@ -126,6 +125,8 @@ class Account::SearchTest < ActiveSupport::TestCase
       ThinkingSphinx::Test.config
       ThinkingSphinx::Test.index verbose: true
 
+      wait_until_index_finished
```

Comment:
How about having a helper that will call
```
module Indexer
  def self.index
    ThinkingSphinx::Test.config
    ThinkingSphinx::Test.index
    wait_until_index_finished
  end
end
```

Reply:
Done